src/inspector/models/popups/abstractelementpopupmodel.cpp

```diff
@@ -23,11 +23,14 @@
 #include "abstractelementpopupmodel.h"
 #include "log.h"
 
+using namespace mu::inspector;
 using namespace mu::notation;
 
 static const QMap<mu::engraving::ElementType, PopupModelType> ELEMENT_POPUP_TYPES = {
     { mu::engraving::ElementType::HARP_DIAGRAM, PopupModelType::TYPE_HARP_DIAGRAM },
     { mu::engraving::ElementType::CAPO, PopupModelType::TYPE_CAPO },
+    { mu::engraving::ElementType::STAFF_TEXT, PopupModelType::TYPE_TEXT },
+    { mu::engraving::ElementType::TEXT, PopupModelType::TYPE_TEXT },
```

Comment:
Sounds good. When will the PR testing take place?

Reply:
It would be great if you could push the fixes for my last round of review comments (and indeed rebase the PR), and then I'll ask someone to start testing!